lib/services/local/export_test.go

```diff
@@ -18,5 +18,12 @@
 
 package local
 
+import "github.com/jonboulle/clockwork"
+
 // SessionDataLimiter exports sdLimiter for tests.
 var SessionDataLimiter = sdLimiter
+
+// SetClock sets the identity service clock for tests.
+func (s *IdentityService) SetClock(clock clockwork.Clock) {
```

Comment:
```suggestion
// SetDesyncedClockForTesting sets its namesake clock in the identity service.
func (s *IdentityService) SetDesyncedClockForTesting(clock clockwork.Clock) {
```

(If you take the other comment.)